### ISSUE TITLE
Issue on 5144426 with multiple-monitor config

### ISSUE THREAD

**Issue by StephenBrown2** [Alexays/Waybar#244]
I just updated waybar using the `waybar-git` package on Arch, and it appears to have broken with multiple monitors configured. Disabling the tray doesn't fix it, but moving my config and letting it use the default (global config for all monitors) works alright.

I updated from [r430.f700319-1 -> r434.5144426-1](https://github.com/Alexays/Waybar/compare/f700319...5144426), so I'm fairly certain it's the vertical placement that's looking for a value that doesn't exist/is one layer deeper.

Message I get when running on the command line:
```shell
❯ waybar
Resources files: /home/stephen/.config/waybar/config, /etc/xdg/waybar/style.css
in Json::Value::resolveReference(key, end): requires objectValue
```

Waybar version:
```shell
❯ waybar --version
Waybar v0.5.0-4-g5144426 (Mar 25 2019, branch 'master')
```

<details><summary>Waybar config:</summary>

```jsonc
[
  {
    "layer": "top",
    "output": "HDMI-A-1",
    "modules-left": ["sway/workspaces", "sway/mode"],
    "modules-center": ["sway/window"],
    "modules-right": [
      "idle_inhibitor",
      "pulseaudio",
      "network",
      "cpu",
      "memory",
      "temperature",
      "clock",
      "tray"
    ],
    "sway/mode": {
      "format": "<span style=\"italic\">{}</span>"
    },
    "idle_inhibitor": {
      "format": "{icon}",
      "format-icons": {
        "activated": "",
        "deactivated": ""
      }
    },
    "tray": {
      "icon-size": 21,
      "spacing": 10
    },
    "clock": {
      "interval": 1,
      "tooltip-format": "{:%c}",
      "format-alt": "{:%Y-%m-%d}",
      "format": "{:%a, %H:%M:%S}"
    },
    "cpu": {
      "format": "{usage}% "
    },
    "memory": {
      "format": "{}% "
    },
    "temperature": {
      "critical-threshold": 80,
      "format": "{temperatureC}°C "
    },
    "network": {
      "format-wifi": "{essid} ({signalStrength}%) ",
      "format-ethernet": "{ifname}: {ipaddr}/{cidr} ",
      "format-disconnected": "Disconnected ⚠"
    },
    "pulseaudio": {
      "scroll-step": 5,
      "format": "{volume}% {icon}",
      "format-bluetooth": "{volume}% {icon}",
      "format-muted": "",
      "format-icons": {
        "headphones": "",
        "handsfree": "",
        "headset": "",
        "phone": "",
        "portable": "",
        "car": "",
        "default": ["", ""]
      },
      "on-click": "pavucontrol"
    }
  },
  {
    "layer": "top",
    "output": "DP-1",
    "modules-left": ["sway/workspaces", "sway/mode"],
    "modules-center": ["sway/window"],
    "modules-right": ["clock"],
    "sway/mode": {
      "format": "<span style=\"italic\">{}</span>"
    },
    "clock": {
      "interval": 60,
      "tooltip-format": "{:%c}",
      "format-alt": "{:%Y-%m-%d}",
      "format": "{:%A, %H:%M}"
    }
  }
]
```
</details>

**Comment by StephenBrown2** [comment]
Confirmed, I commented out the array and second monitor config in my config file and waybar loaded up fine for the one monitor:
```
❯ waybar
Resources files: /home/stephen/.config/waybar/config, /etc/xdg/waybar/style.css
Bar configured (width: 3072, height: 30) for output: HDMI-A-1
```

<details><summary>Commented (working) config:</summary>

```jsonc
// [
  {
    "layer": "top",
    "output": "HDMI-A-1",
    "modules-left": ["sway/workspaces", "sway/mode"],
    "modules-center": ["sway/window"],
    "modules-right": [
      "idle_inhibitor",
      "pulseaudio",
      "network",
      "cpu",
      "memory",
      "temperature",
      "clock" //,
      // "tray"
    ],
    "sway/mode": {
      "format": "<span style=\"italic\">{}</span>"
    },
    "idle_inhibitor": {
      "format": "{icon}",
      "format-icons": {
        "activated": "",
        "deactivated": ""
      }
    },
    "tray": {
      "icon-size": 21,
      "spacing": 10
    },
    "clock": {
      "interval": 1,
      "tooltip-format": "{:%c}",
      "format-alt": "{:%Y-%m-%d}",
      "format": "{:%a, %H:%M:%S}"
    },
    "cpu": {
      "format": "{usage}% "
    },
    "memory": {
      "format": "{}% "
    },
    "temperature": {
      "critical-threshold": 80,
      "format": "{temperatureC}°C "
    },
    "network": {
      "format-wifi": "{essid} ({signalStrength}%) ",
      "format-ethernet": "{ifname}: {ipaddr}/{cidr} ",
      "format-disconnected": "Disconnected ⚠"
    },
    "pulseaudio": {
      "scroll-step": 5,
      "format": "{volume}% {icon}",
      "format-bluetooth": "{volume}% {icon}",
      "format-muted": "",
      "format-icons": {
        "headphones": "",
        "handsfree": "",
        "headset": "",
        "phone": "",
        "portable": "",
        "car": "",
        "default": ["", ""]
      },
      "on-click": "pavucontrol"
    }
  } //,
//   {
//     "layer": "top",
//     "output": "DP-1",
//     "modules-left": ["sway/workspaces", "sway/mode"],
//     "modules-center": ["sway/window"],
//     "modules-right": ["clock"],
//     "sway/mode": {
//       "format": "<span style=\"italic\">{}</span>"
//     },
//     "clock": {
//       "interval": 60,
//       "tooltip-format": "{:%c}",
//       "format-alt": "{:%Y-%m-%d}",
//       "format": "{:%A, %H:%M}"
//     }
//   }
// ]
```
</details>

**Comment by Alexays** [comment]
Fixed with dda0cc793e4f7b6cc2660fc4c53a39f41a524fa2

**Comment by StephenBrown2** [comment]
Fix confirmed! Thanks for the rapid response!
```shell
❯ waybar    
Resources files: /home/stephen/.config/waybar/config, /etc/xdg/waybar/style.css
Tray is in beta, so there may be bugs or even be unusable.
Bar configured (width: 1050, height: 30) for output: DP-1
Bar configured (width: 3072, height: 30) for output: HDMI-A-1
```
